projects/storefrontstyles/scss/components/misc/popover/_popover.scss

```diff
@@ -0,0 +1,174 @@
+$cx-popover-font-size: 0.875rem !default;
```

Comment:
When setting up variables we should follow the css-architecture guidelines in the Spartacus documentation found [here](https://sap.github.io/spartacus-docs/css-architecture/)
`:root {
  --cx-color-primary: red;
}

cx-link {
  color: var(--cx-color-primary);
}`

Reply:
Updated